include/mbedtls/mbedtls_config.h

```diff
@@ -1707,6 +1707,16 @@
  */
 //#define MBEDTLS_THREADING_PTHREAD
 
+/**
+ * \def MBEDTLS_TRACE
+ *
+ * Enable the trace functions to report invocations of various primitives.
+ * See 'include/mbedtls/trace.h' for more information.
+ *
+ * Uncomment to enable trace output.
+ */
+#define MBEDTLS_TRACE
```

Comment:
This needs to be disabled by default. With it enabled, the logs are so large that our CI kills them.

Testing with traces enabled may need to have stderr redirected to a file, to avoid large console logs (but the files will also have a size limit). To be investigated.

Reply:
Fixed. Thanks.